README.md

```diff
@@ -48,22 +48,30 @@ dotnet build --no-incremental /p:WarningsAsErrors=true
 
 ## Database Migrations
 
-First initialize the settings file by copying `appsettings.example.json` as `appsettings.Development.json` and providing
+***Important***: Initialize the settings file in the `src/Aquifer.Migrations` directory by copying `appsettings.example.json` as `appsettings.Development.json` and providing
 the values needed for your instance.
 
 Entity Framework will generate migrations by comparing the C# Entities defined
 in the project and the current state of the database.
 
-To create a new migration:
+### Create a New Migration
```

Comment:
We may want to call this `### Adding a New Entity/Table` and differentiate from the process of the migration itself. Since I could just update a column on an existing table without needing to mess with the AquiferDbContext. Maybe below could be `### Create a New Migration`?